src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export enum RequestMethod {
+    CheckConnection = 'checkConnection',
+    Connect = 'connect',
+    Disconnect = 'disconnect',
+    UnsubscribeFromBalance = 'unsubscribeFromBalance',
+    UnsubscribeFromIsConnected = 'unsubscribeFromIsConnected',
```

Comment:
I still think that this should be private in the extension itself (details in PMs)